### PR TITLE
Clarify the expected format of round IDs. Resolves #203

### LINKED ISSUE #203: Clarify the expected format of round IDs
As there's been regular confusion and repeated associated discussion on this topic, e.g.:
- https://github.com/orgs/hubverse-org/discussions/10
- https://github.com/hubverse-org/hubValidations/issues/81
- https://github.com/hubverse-org/hubValidations/discussions/13
- https://github.com/orgs/hubverse-org/discussions/12

it would be good to clarify the expectation in the docs.

Round IDs conform to one of the following patterns:
1. an ISO date
2. any combination of letters, numbers or hyphen (`_`) (but also consider: https://github.com/hubverse-org/hubValidations/issues/135)

This might be the ideal initial section for this: https://hubverse.io/en/latest/user-guide/model-output.html#directory-structure
but open to suggestions on sections it might be worth repeating in.

### PR DESCRIPTION
**Pull request by annakrystalli**
This PR is the first step in clarifying round ID format expectation. Resolves #203 

1. Added explicit clarification of round_id format expectations (as well as all the other elements)
2. Made the directory structure section clearer 
3. Added realistic model output directory examples that includes example alphanumeric `round_id`s

### CODE REVIEW

File: docs/source/user-guide/model-output.md [hunk 1]
```diff
@@ -4,23 +4,53 @@
...
+* `model_id3`
+   * `<round_id1>-<model_id3>.csv` (or parquet, etc)
 
 where `model_id` = `team_abbr-model_abbr`
 
+### Expected patterns
+
+The elements making up model output directory and file names must match the following patterns:
+
+- `round_id`s must be either ISO fomatted dates (YYYY-MM-DD) or any combinations of alphanumerics separated by underscores (`_`).
```
Comment (elray1) [inline docs/source/user-guide/model-output.md:21]:
```suggestion
- `round_id`s must be either ISO fomatted dates (YYYY-MM-DD) or any combination of alphanumerics separated by underscores (`_`).
```

File: docs/source/user-guide/model-output.md [hunk 2]
```diff
@@ -4,23 +4,53 @@
...
+- `team_abbr` and `model_abbr` must contain any combinations of alphanumerics separated by underscores (`_`).
```
Comment (elray1) [inline docs/source/user-guide/model-output.md:22]:
```suggestion
- `team_abbr` and `model_abbr` must contain any combination of alphanumerics separated by underscores (`_`).
```

File: docs/source/user-guide/model-output.md [hunk 3]
```diff
@@ -4,23 +4,53 @@
...
+
+* `hub-baseline`
+   * `2024_2025_1_covid-hub-baseline.csv`
+   * `2024_2025_1_flu-hub-baseline.csv`
+* `team_1-ensemble`
+   * `2024_2025_1_covid-team_1-ensemble.parquet`
+   * `2024_2025_1_flu-team_1-ensemble.gz.parquet` 
+
+### Example directory structure and file names
+
```
Comment (elray1) [inline docs/source/user-guide/model-output.md:53]: This section header seems to be extra?
```suggestion
```

**Review by elray1 — CHANGES_REQUESTED** [review]

**Review by elray1 — APPROVED** [review]

File: docs/source/user-guide/model-output.md [hunk 4]
```diff
@@ -4,23 +4,53 @@
...
 
 where `model_id` = `team_abbr-model_abbr`
 
+### Expected patterns
+
+The elements making up model output directory and file names must match the following patterns:
+
+- `round_id`s must be either ISO fomatted dates (YYYY-MM-DD) or any combinations of alphanumerics separated by underscores (`_`).
+- `team_abbr` and `model_abbr` must contain any combinations of alphanumerics separated by underscores (`_`).
+- `model_id`s are composed of `team_abbr` and `model_abbr` separated by a hyphen (i.e. `team_abbr-model_abbr`).
```
[the remark this answers is not in the document]
Comment (annakrystalli) [reply docs/source/user-guide/model-output.md]: Reversed in [a79409a](https://github.com/hubverse-org/hubDocs/pull/225/commits/a79409ac0b600ff85774d04f5f0ec177470d2ad2)

**Pull request merged**